refresh_apartment/views.py

```diff
@@ -148,7 +148,13 @@ def bckgrndfetch(city_list, limit):
                     continue
 
                 # match the not null constraint of postgre
-                if state is None or address is None or city is None or zipcode is None:
```

Comment:
How about `if not State or not address or not city ...:`?
I'm not 100% sure but this might check if it's None or is an empty string.

Reply:
@alldne Updated!

Reply:
Hmm what I meant was
```
 if (
                    not state
                    or not address
                    or not city
                    or not zipcode
                    or not locality
```

Isn't it equivalent in terms of functionality and more concise?

Reply:
@alldne Oh, it is! The python way of string comparison.
Updated.